R/rocRAND/build_tarballs.jl

```diff
@@ -51,11 +55,11 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    BuildDependency(PackageSpec(; name="LLVM_full_jll", version=v"11.0.1")),
+    BuildDependency(PackageSpec(; name="ROCmLLVM_jll", version=v"4.2.0")),
     Dependency("hsa_rocr_jll"),
     Dependency("ROCmCompilerSupport_jll"),
     Dependency("ROCmOpenCLRuntime_jll"),
-    Dependency("HIP_jll"),
+    Dependency("HIP_jll", v"4.2.0"),
```

Comment:
If the version is important also for runtime you may want to specify the compat bound as well:
```suggestion
    Dependency("HIP_jll"; compat="4.2.0"),
```